### PR TITLE
Updating my progres of get validation 

### PR DESCRIPTION
**Pull request by sperBruno**
I'm till working on this please do not marge yet, thanks.

### CODE REVIEW

**Comment by sperBruno** [comment]
Now you can marge my code with this last update.

File: Framework7/features/feature_definitions/get_comment_feature/get_comments.feature [hunk 1]
```diff
@@ -1,8 +1,62 @@
...
\ No newline at end of file
+    Then I expect Status code 200
+
+  @get_comment @get_comment @smoke
+  Scenario: getting individual comment request from a User Story
+
+    When I send a GET request to /projects/1596603/stories/120320037/comments/135075719
+    Then I expect Status code 200
+
+  @validating_ID_value  @story_id @person_id @get_comment @functional
```
Comment (wMercadoJ) [inline Framework7/features/feature_definitions/get_comment_feature/get_comments.feature:19]: Think in use single tags @comment @get @functional

File: Framework7/features/feature_definitions/get_comment_feature/get_comments.feature [hunk 2]
```diff
@@ -1,8 +1,62 @@
...
+      | person_id         |
+      | story_id          |
+
+  @validating_json_has_7_parameters @get_comment
+  Scenario: verifying that a comment has seven parameters
+    When I send a GET request to /projects/1596603/stories/120320037/comments/135075719
+    And I validate that the json comment has 7 parameters
+    Then I should receive true as an answer when the json has 7 numbers
+
+  @validating_Kind_value @validating_text_value @get_comment @functional @comment_has_2000_characters
```
Comment (wMercadoJ) [inline Framework7/features/feature_definitions/get_comment_feature/get_comments.feature:37]: too complex tags

File: Framework7/src/requests/validate.rb [hunk 3]
```diff
@@ -1,14 +1,32 @@
...
+  def text_is_between_a_range(text, min_length_character, max_length_character)
+    if text.length >= min_length_character && text.length <= max_length_character
+      true
+    else
+      false
+    end
+  end
+
+  def is_a_date(date)
+    if date =~ (/^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T(0[0-9]|[1-5][0-9]):(0[0-9]|[1-5][0-9]):(0[0-9]|[1-5][0-9])Z$/)
```
Comment (wMercadoJ) [inline Framework7/src/requests/validate.rb:27]: Maybe you can move the Regex into a Constant in base class in order that everybody can use it

**Pull request merged**